terraform-module-standards.md

```diff
@@ -345,14 +345,19 @@ You should make updating the version pin a regular practice:
 ```hcl
 provider "equinix" {
   source = "equinix/equinix"
-  version = "= 1.11.0-alpha.2"
+  version = "= 1.11.0"
 }
 ```
 
 In shared modules, the provider version should **not** be pinned. Instead, a [version constraint](https://developer.hashicorp.com/terraform/language/providers/requirements#best-practices-for-provider-versions) should be added targeting the minimum working minor version. For example, using `~>1.11` indicates the module will work with any provider version `>= 1.11, < 2.0`.
 
 ```hcl
 terraform { 
+  provider_meta "equinix" {
+    # Set the name of the module below 
+    module_name = "equinix-labs"
```

Comment:
Names are hard. "equinix-labs" seems fine since it is an example that shows the point of the comment `# Set the name of the module below`.  I'm not worried that we'll end up with dozens of projects reporting themselves as `equinix-labs` because this is documentation more than code, and I believe users will be more thoughtfully copying and pasting this rather than forking and forgetting (to update it).